mcs/class/System.XML/ReferenceSources/BinaryCompatibility.cs

```diff
@@ -0,0 +1,7 @@
+namespace System.Xml
+{
+	static class BinaryCompatibility
+	{
+		public static bool TargetsAtLeast_Desktop_V4_5_2 => true;
```

Comment:
This could be a readonly field instead of a property, it was only made a property because the reflection magic from referencesource looked for the property in corlib:

https://github.com/mono/mono/blob/9418d57c13b27ec59269762d54afd5fe596e017f/mcs/class/corlib/ReferenceSources/BinaryCompatibility.cs#L7-L8

^ we can get rid of that one now.